index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin.<br />Glad to see<br />
         </li>
     </ul>
 </div>
-</section>
\ No newline at end of file
+</section>
+<!--Works Section page 1-->
+<section class="page1">
+    <div class="@continer">
+  
+      <div class="layout1">
+        <a href="#">
+          <img
+            class="imagework"
+            src="image/works/Snapshoot Portfolio.png"
+            alt="background"
+          />
+        </a>
+        <div class="pra">
+          <h3 class="title1">Tonic</h3>
+          <ul class="flex">
+            <li class="canopy-">CANOPY</li>
+            <li class="p"> <img src="image/works/Counter.png"></li>
+            <li class="role">Back End Dev</li>
+            <li class="p"> <img src="image/works/Counter.png"></li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="text1">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="btn1">
+            <li class="btn">
+              <button class="html" type="button">HTML</button>
+            </li>
+            <li class="btn">
+              <button class="css" type="button">CSS</button>
+            </li>
+            <li class="btn">
+              <button class="java" type="button">JAVASCRIPT</button>
+            </li>
+          </ul>
+          <div class="btn2">
+            <button type="button" class="btn-see">See Project</button>
+          </div>
+        </div>
+      </div>
+  <!--Works Section page 2-->
+<section class="page2">
```

Comment:
Please update your webpage to use `<div>` tags to wrap each project card instead of `<section>` tags. The section tag should be used to wrap different parts of your webpage such as headline, works, sections etc as opposed to different project cards.